### PR TITLE
fix check if tasty is installed in a proper path

### PR DESCRIPTION
**Pull request by danielchg**

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -63,24 +70,35 @@ func init() {
...
 	err := os.Symlink(execFile, execOcLink)
 	if err != nil {
-		color.Yellow("Oc Plugin already installed.")
+		if strings.Contains(err.Error(), "file exists") {
+			found = true
+		} else {
+			log.Fatal(err)
+		}
+	} else {
 		found = true
```
Comment (karmab) [inline cmd/config.go:86]: found shouldn't be set to True when your code actually creates the link

File: cmd/config.go [hunk 2]
```diff
@@ -63,24 +70,35 @@ func init() {
...
 	err = os.Symlink(execFile, execKubectlLink)
 	if err != nil {
-		color.Yellow("Kubectl Plugin already installed.")
+		if strings.Contains(err.Error(), "file exists") {
+			found = true
+		} else {
+			log.Fatal(err)
+		}
+	} else {
 		found = true
```
Comment (karmab) [inline cmd/config.go:98]: same comment as before

**Review by karmab — CHANGES_REQUESTED** [review]

File: cmd/config.go (hunk 1 above)
Comment (danielchg) [reply cmd/config.go:86]: If found is not set to true the log messafe of Already installed is not shown

Comment (karmab) [reply cmd/config.go:86]:
```
$ which oc-olm
/xxx/bin/oc-olm
$  which kubectl-olm
/xxx/bin/kubectl-olm
$  rm /xxx/bin/kubectl-olm
$ tasty config -p
Installing tasty as kubectl and oc CLI plugin
Tasty already installed as oc and kubectl plugin, you can try oc olm --help or kubeclt olm --help
```

Comment (danielchg) [reply cmd/config.go:86]: the thing is that if we want to say something when is already installed and when de installation has finished successfully, but if we remove the found true and is the first time that is installed will not show any message to the user

Comment (danielchg) [reply cmd/config.go]: check it out with the new changes

**Pull request merged**